crates/common/src/signing.rs

```diff
@@ -156,7 +156,9 @@ impl DirectoryStoredKeys {
         debug!(generate_secret_at = ?secretpath);
         let secret = SecretKey::random(StdRng::from_entropy());
 
-        let privpem = secret.to_pkcs8_pem(LineEnding::CRLF)?;
+        let privpem = secret
+            .to_pkcs8_pem(LineEnding::CRLF)
+            .map_err(|_| SignerError::InvalidPrivateKey {})?;
```

Comment:
Don't know if we might want to log these `_` but not a blocker for this PR.

Reply:
I think, it's a breaking change we did on main